src/client/App.jsx

```diff
@@ -1,30 +1,59 @@
-import { useEffect, useState } from 'react';
-import './App.css';
-import MovieForm from './components/MovieForm';
-import UserForm from './components/UserForm';
+import { useEffect, useState } from "react";
```

Comment:
i am not sure where/how to comment on commits but i think it would be good practice to have made more commits and some detailing on each commit. I guess it might be safer for the work. but overall good project man